### PR TITLE
Header Section

### PR DESCRIPTION
**Pull request by kylecalbert**
- Created a resuable button which will be used for the header section
- Created a resuable text component

### CODE REVIEW

File: ticket-booker/src/components/Header/HeaderButton.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        </LeftContainerContent>
+        <Text margin={'0 10px 0 0'}>6</Text>
+      </ButtonContainer>
+      <ButtonContainer>
+        <LeftContainerContent>
+          <IconContainer src={TicketsIcon} />
+          <Text margin={'0 0 0 10px'}>Concerts</Text>
+        </LeftContainerContent>
+        <Text margin={'0 10px 0 0'}>6</Text>
+      </ButtonContainer>
```
Comment (wingleungchoi) [inline ticket-booker/src/components/Header/HeaderButton.js:50]: each ButtonContainer looks similar.
Maybe you can create a new component and use map to loop an array to return them?
```
[{text1: "All trickets", text2: "6"}, ..].map(({text1, text2}) => {
<NewComponent text1="text1", text2="text2">
})

```

File: ticket-booker/src/components/Header/Header.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { HeaderContainer, TextContainer } from './Header.style';
+import { darkModeColors } from '../../constants/colors';
```
Comment (wingleungchoi) [inline ticket-booker/src/components/Header/Header.js:3]: would you consider to implement a functionality to switch between dark and light mode?

Comment (kylecalbert) [reply ticket-booker/src/components/Header/Header.js:3]: Yes, that will be the aim for  a future sprint

File: ticket-booker/src/components/Header/HeaderButton.js (hunk 1 above)
Comment (kylecalbert) [reply ticket-booker/src/components/Header/HeaderButton.js:50]: Will implement this idea in future sprint

**Pull request merged**